src/serialized-analysis.ts

```diff
@@ -44,7 +44,10 @@ export interface SourceLocation {
 }
 
 export interface AnalyzedPackage {
```

Comment:
how about renaming this file and AnalyzedPackage to be elements-metadats specific. I don't think it should be viewed as a generic serialization of an analysis - it's a document format that could be produced by hand or another tool, and we happen to be building support for it into this package. In fact, we might want to publish the schema in a separate project.